### ISSUE TITLE
Custom actions doesn't work

### ISSUE THREAD

**Issue by Icaruk** [mingo-app/mingo#626]
**Describe the bug**
Custom actions doesn't work.
I've used it some time ago, now I can't.

Here is the part of the template Mingo gives you:
```js
async function run() {
    await projects.updateOne(
      {_id: CurrentDocument._id},
      {$set: {
         field: null
      }}
    );
    return 'Done';
  }
```

**Screenshots**
![image](https://github.com/mingo-app/mingo/assets/10779469/35b82f52-fcd6-4512-a219-095e46d09cee)

**Desktop (please complete the following information):**
 - Mingo version: 1.13.5
 - OS: MacOS Sonoma 14.2

**Comment by tothradoslav** [comment]
Thanks for this, corrected. Now you can access collections directly by their names (case sensitive), such as in the example or by using "db.collection(name)"

**Comment by tothradoslav** [comment]
This issue is fixed / available in the new release v1.14.1

https://github.com/mingo-app/mingo/releases/tag/v1.14.1

Please reopen this issue if necessary.
